docs/reference/components.cue

```diff
@@ -487,14 +492,408 @@ components: {
+				default_namespace: "vector"
+				tags:              _component_tags
+			}
+			_request_read_errors_total: {
+				description:       "The total number of request read errors for this component."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _component_tags
+			}
+			_requests_completed_total: {
+				description:       "The total number of requests completed by this component."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _component_tags
+			}
+			_requests_received_total: {
+				description:       "The total number of requests received by this component."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _component_tags
+			}
+			_timestamp_parse_errors_total: {
+				description:       "The total number of errors encountered parsing [RFC3339](\(urls.rfc_3339)) timestamps."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _component_tags
+			}
+			_uptime_seconds: {
+				description:       "The total number of seconds the Vector instance has been up."
+				type:              "gauge"
+				default_namespace: "vector"
+				tags:              _component_tags
+			}
+
+			// Vector instance metrics
+			_config_load_errors_total: {
+				description:       "The total number of errors loading the Vector configuration."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_connection_errors_total: {
+				description:       "The total number of connection errors for this Vector instance."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_quit_total: {
+				description:       "The total number of times the Vector instance has quit."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_recover_errors_total: {
+				description:       "The total number of errors caused by Vector failing to recover from a failed reload."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_reload_errors_total: {
+				description:       "The total number of errors encountered when reloading Vector."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_reloaded_total: {
+				description:       "The total number of times the Vector instance has been reloaded."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_started_total: {
+				description:       "The total number of times the Vector instance has been started."
+				type:              "counter"
+				default_namespace: "vector"
+				tags:              _internal_metrics_tags
+			}
+			_stopped_total: {
```

Comment:
There are duplicates for this in `internal_metrics.cue` under reloaded_total, started_total,  stopped_total, and others. Maybe this ones should removed.